examples/surface_timeseries_.py

```diff
@@ -14,6 +20,14 @@
         "You must have nilearn installed to run this example."
     ) from None
 
+import numpy as np
+
+if parse(np.__version__) >= parse("1.24"):
+    raise RuntimeError(
+        "Incompatible numpy version. "
+        "You must have numpy <1.24 for nilearn to work and download the example data"
```

Comment:
Two suggestions:
1. Replace the `<` symbol with with the words "less than" (because people skim read, and very often these types of error messages are encouraging people to *upgrade* their versions, not downgrade like we are asking them to do here)
2. Write down which version(s) of nilearn this is currently true for, i.e. nilearn version 0.10.1 and below (because this info is hard to find easily the more time has gone by, and possibly newer version s of nilearn that are compatible might be released)

Reply:
A nilearn PR that could fix this problem is open here: https://github.com/nilearn/nilearn/pull/3644
It's still under development but once a fix is in and a new version published, we'll probably require that as the minimum nilearn version (and remove the numpy version warning) for this example script.